Cargo.toml

```diff
@@ -13,7 +13,7 @@ documentation = "https://docs.rs/tracing-glog"
 tracing = { version = "0.1", default-features = false }
 tracing-subscriber = { version = "0.3.3", features = ["std", "fmt", "registry", "ansi", "time", "local-time"], default-features = false }
```

Comment:
since we're making nu-ansi-term optional, we should make on `tracing-subscriber`'s `ansi` feature dependent on `tracing-glog`'s `ansi` feature being set through `https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies`.

(an msrv of 1.60 is not an issue here—the MSRV for this crate is a de jure "whatever the most recent stable Rust is".)